wandb/sdk/artifacts/artifact.py

```diff
@@ -1138,13 +1154,22 @@ def add_file(
                 to the basename of the file.
             is_tmp: If true, then the file is renamed deterministically to avoid
                 collisions.
+            skip_cache: If set to `True`, W&B will not copy/move files to the cache while uploading
+            policy: "mutable" | "immutable". By default, "mutable"
+                "mutable": Copy to staging
+                    - If `skip_cache` = True: Delete staged files after uploading
+                    - If `skip_cache` = False: Move to cache after uploading
+                "immutable": Require file to not change.
+                    - If `skip_cache` = True: Do nothing.
+                    - If `skip_cache` = False: Copy to cache synchronously.
```

Comment:
```suggestion
                "mutable": Create a temporary copy of the file to prevent corruption during upload.
                "immutable": Disable protection, rely on the user not to delete or change the file.
```
This should provide a description that helps users decide which option to pick, and importantly, should be an option that is independent of whether to skip caching.